games-roguelike/FTL-gog/FTL-gog-1.6.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop unpacker
+
+DESCRIPTION="Top-down roguelike space ship simulator"
+HOMEPAGE="https://www.gog.com/game/faster_than_light"
+SRC_URI="ftl_advanced_edition_en_1_6_3_17917.sh"
+RESTRICT="bindist fetch mirror strip test"
+
+LICENSE="GOG-EULA"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE=""
+
+DEPEND="app-arch/unzip"
+RDEPEND="media-libs/freetype
+	media-libs/libsdl[X,sound,opengl,video]
+	media-libs/libpng
+	sys-libs/zlib
+	virtual/opengl"
+
+S="${WORKDIR}/data/noarch"
+
+pkg_nofetch() {
+	elog "Please buy and download ${SRC_URI} from"
+	elog "https://www.gog.com/game/faster_than_light"
+	elog "and copy it to ${DISTDIR}"
+}
+
+src_unpack() {
+	unpack_zip "${DISTDIR}/${A}"
+}
+
+src_install() {
+	insinto /opt/gog/FTL
+	doins -r *
+	fperms +x start.sh game/FTL game/data/FTL*
```

Comment:
You need to specify the full path to the files (e.g. `/opt/gog/FTL/start.sh`). Otherwise, this will change the permissions of the files in `${S}`.

Reply:
And I recommend putting the path in a local (e.g. `local dir="/opt/gog/FTL"`), so that you don't repeat yourself and can easily change it in one place.